packages/apps/spaces/app/src/components/RootSidenav/RootSidenav.tsx

```diff
@@ -9,23 +11,23 @@ import { Image } from '@ui/media/Image';
 import { VStack } from '@ui/layout/Stack';
 import { GridItem } from '@ui/layout/Grid';
 
-import { Box } from '@ui/layout/Box';
-import { useLocalStorage } from 'usehooks-ts';
 import { useJune } from 'app/src/hooks/useJune';
 import { LogOut01 } from '@ui/media/icons/LogOut01';
 import { getGraphQLClient } from '@shared/util/getGraphQLClient';
+import { useOrganizationsMeta } from '@shared/state/OrganizationsMeta.atom';
 import { useGlobalCacheQuery } from '@shared/graphql/global_Cache.generated';
 
+import logo from './customer-os.png';
```

Comment:
this could move it to public folder

Reply:
About this, I think it loads better and has blur placeholder if we load it this way.